shimmingtoolbox/cli/b0shim.py

```diff
@@ -74,10 +74,19 @@ def b0shim_cli():
               help="Regularization factor for the current when optimizing. A higher coefficient will penalize higher "
                    "current values while 0 provides no regularization. Not relevant for 'pseudo-inverse' "
                    "optimizer_method.")
-@click.option('--optimizer-criteria', 'opt_criteria', type=click.Choice(['mse', 'mae']), required=False,
+########################################################################################################################
+@click.option('--weighting-signal-loss', 'w_signal_loss', type=click.FLOAT, required=False, default=0.0, show_default=True,
+              help="weighting for signal loss recovery. Since there is generally a compromise between B0 inhomogeneity"
+              " and signal loss recovery, a higher coefficient will put more weights to recover the signal loss over "
+              "the B0 inhomogeneity.")
+@click.option('--epi_echo_time', 'epi_te', type=click.FLOAT, required=False, default=0.0, show_default=True,
```

Comment:
Same comment as above

Reply:
Thanks! Addressed in https://github.com/shimming-toolbox/shimming-toolbox/commit/9c2ba526bef3781adfbd5b34d95d37b3e7f5c7da